colossalai/fx/passes/meta_info_prop.py

```diff
@@ -150,7 +151,7 @@ def get_attr(self, target: 'Target', args: Tuple[Argument, ...], kwargs: Dict[st
             flop_count (Tuple): The flop count for (fwd_flop, bwd_flop).
             mem_stat (Tuple): The memory statistics for (fwd_tmp, fwd_out, bwd_tmp, bwd_out)
         """
-        return super().get_attr(target, args, kwargs), (0, 0), (0, 0, 0, 0)
+        return super().get_attr(target, args, kwargs), {}
 
     @compatibility(is_backward_compatible=True)
     def call_function(self, target: 'Target', args: Tuple[Argument, ...], kwargs: Dict[str, Any]) -> Any:
```

Comment:
The return type is not Any, this is the same for other functions. Check the type hinting again.

Reply:
I will return a new `MetaInfo` dataclass.